[imagezipview] Improve ZIP multi-image viewer styles and add controls

ahd44 opened the pull request:
Improve imagezipview by styling elements with Bootstrap and adding controls to easily change between images, improving the layout and making it more functional for users.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1209524177992855

jonathanthiry commented on src/main/resources/frontendapps/imagezipview/imagezipview.html:
```diff
@@ -1,18 +1,29 @@
+    <div class="m-2">
+      <div class="d-flex mb-2 align-items-center">
+        <button title="To previous image" type="button" id="previous-button"
+            class="btn btn-outline-secondary me-2" style="width:50px; height:50px;" disabled="">
```
Not sure about the 50px, it breaks the visual consistency next to the other buttons on the page.

ahd44 replied:
Yes, it is not needed and arbitrary... will try to avoid that in the future.

jonathanthiry reviewed (commented):
Great improvement! Two things:
- It would be nice to be able to use the keyboard navigation from the portal page.
- We could remove the extra white space on the right of the portal embed. It would be to wrap the iframe in a `col` inside the `row`, and probably remove the `width: 100%`.
- It would be nice to update the URL when we navigate images, so that someone can send a link to a specific image. This could also be done in a separate task.

ahd44 commented on src/main/resources/frontendapps/imagezipview/imagezipview.js:
Left a single comment that explains why a local variable was used over directly using selectedIndex

ahd44 commented:
Fixed in-line comments, also fixed a bug when using left/right arrows with the select element focused, where it would skip images then.

For the [general comments](https://github.com/ICOS-Carbon-Portal/data/pull/341#pullrequestreview-2661292019), we can either delay this PR until we can make those changes or go ahead with these changes and make a new task for integrating this better with the portal overall.
